src/main/js/component/channels/EmailConfiguration.js

```diff
@@ -29,20 +29,53 @@ class EmailConfiguration extends React.Component {
         if (nextProps.updateStatus === 'FETCHED' || nextProps.updateStatus === 'UPDATED') {
             this.setState({
                 mailSmtpHost: nextProps.mailSmtpHost || '',
-                mailSmtpFrom: nextProps.mailSmtpFrom || '',
-                mailSmtpAuth: nextProps.mailSmtpAuth || false,
                 mailSmtpUser: nextProps.mailSmtpUser || '',
                 mailSmtpPassword: nextProps.mailSmtpPassword || '',
                 mailSmtpPasswordIsSet: nextProps.mailSmtpPasswordIsSet || false,
                 mailSmtpPort: nextProps.mailSmtpPort || undefined,
```

Comment:
why undefined?  if the values don't exist they'd be undefined anyway.  A value like 0 would evaluate to false and the boolean values should already be true/false?

Reply:
I think early on the values were strings that's probably why.  let me fix that.